B827EBFFFE9F7CA8.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "the id as you noted down   in the install.sh console output", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-gateway", 
```

Comment:
and here it's not the address of the gateway but of the router, please check the list of available routers: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html